classes/mock/generator.php

```diff
@@ -201,6 +201,26 @@ public function generate($class, $mockNamespace = null, $mockClass = null)
 
 		return $this;
 	}
+	
+	public function getNewMockInstance($class, $mockNamespace = null, $mockClass = null, array $constructorArguments = array())
```

Comment:
I would remove this method as the purpose of the mock generator is to generate mock classes, not instances.